collection-scripts/gather_haproxy_config

```diff
@@ -11,7 +11,7 @@ function gather_haproxy_config {
     for IC in ${INGRESS_CONTROLLERS}; do
         PODS=$(oc get pods -n openshift-ingress --no-headers -o custom-columns=":metadata.name" --selector ingresscontroller.operator.openshift.io/deployment-ingresscontroller="${IC}")
         for POD in ${PODS}; do
-            oc cp openshift-ingress/"${POD}":/var/lib/haproxy/conf/haproxy.config "${HAPROXY_CONFIG_PATH}"/"${IC}"/"${POD}"/haproxy.config &
+            timeout -v 3m oc cp openshift-ingress/"${POD}":/var/lib/haproxy/conf/haproxy.config "${HAPROXY_CONFIG_PATH}"/"${IC}"/"${POD}"/haproxy.config &
```

Comment:
Is a 3 min timeout really how long we want to wait with this? 
This copy operation should be seconds; IE 10x (seconds) is likely a 1-1.5 m wait time (can we half this)?

Reply:
https://coreos.slack.com/archives/CFDM5CQMN/p1657143478060469?thread_ts=1656698903.916469&cid=CFDM5CQMN discussed this. 

/lgtm